src/cache/lru_cache_manager/lru_queue.h

```diff
@@ -21,15 +22,16 @@ typedef struct LRUQueue {
   LRUNode *tail;            // Queue tail
   LRUNode *emptySpace;      // Next empty place in the queue
   bool fullCapacity;        // Indication if the queue was reached full cacpcity
+  LRUNode **emptyCells;
+  cacheValueFreeFunc freeCB;
```

Comment:
I think this callback should be a part of the cache object.

Reply:
actual free is in the queue